SUMMARY.md

```diff
@@ -48,6 +48,7 @@
   * [Adobe Campaign Standard event reference](using-mobile-extensions/adobe-campaign-standard/adobe-campaign-standard-event-reference.md)
 * [Adobe Target](using-mobile-extensions/adobe-target/README.md)
   * [Target API reference](using-mobile-extensions/adobe-target/target-api-reference.md)
```

Comment:
Can we change the title to "[Target API reference (new)]" to match the deprecated item?

Reply:
Sure, done